apps/test-suite/TestUtils.js

```diff
@@ -38,6 +38,7 @@ export function getTestModules() {
       modules.push(require('./tests/GLView'));
     }
 
+    // Pending https://github.com/expo/expo-cli/pull/940
```

Comment:
Remove that line since https://github.com/expo/expo-cli/pull/940 is merged?

Reply:
not published yet